### ISSUE TITLE
model

### ISSUE THREAD

**Issue by wcpcp** [antifen/Dialation-Supervision#2]
1.i find that the model in the paper has the adaptive threshold network,but in the code not use.
2.the loss function in the code seems to have some difference from the paper's

**Comment by antifen** [comment]
1. For the loss function:


Paper III. A has clarified that our adaptive threshold selection network is independent of the previous two-stage segmentation network and trained independently. The loss function provided in the paper only elaborates on the loss functions of the first two stages of the network, and does not include the loss of adaptive threshold selection network training. For the loss functions in the first two stages, the l1 parameter and dice loss in the code are different from those in the paper. This is our attempt during parameter debugging, and the optimal choice is still as described in the paper.




2. For adaptive threshold selection networks:


The training of the adaptive threshold selection network is independent of the first two stages of network training, which are respectively trained in "optimizer_net_adaptive_threshol" of "train_vessel.py", and code annotation of "crop_eval.py"  in "eval_test.py". As for the detailed structure of the adaptive threshold selection network, we will immediately announce it later. You can reproduce it yourself as described in the paper, or try using a modified classification network instead.




Thank you for following our work and asking questions.

---Original---
From: ***@***.***&gt;
Date: Sat, Oct 21, 2023 17:16 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [antifen/Dialation-Supervision] model (Issue #2)




 
1.i find that the model in the paper has the adaptive threshold network,but in the code not use.
 2.the loss function in the code seems to have some difference from the paper's
